src/System.Private.CoreLib/src/System/Activator.cs

```diff
@@ -62,6 +64,169 @@ internal static object CreateInstance(Type type, bool nonPublic, bool wrapExcept
+
+            if (o == null)
+                return null;
+            else
+            {
+                ObjectHandle Handle = new ObjectHandle(o);
+                return Handle;
+            }
+        }
+
+        public static ObjectHandle CreateInstanceFrom(string assemblyFile, string typeName)
+        {
+            return CreateInstanceFrom(assemblyFile, typeName, null);
+        }
+
+        public static ObjectHandle CreateInstanceFrom(string assemblyFile, string typeName, bool ignoreCase, BindingFlags bindingAttr, Binder binder, object[] args, CultureInfo culture, object[] activationAttributes)
+        {
+            return CreateInstanceFromInternal(assemblyFile,
+                                              typeName,
+                                              ignoreCase,
+                                              bindingAttr,
+                                              binder,
+                                              args,
+                                              culture,
+                                              activationAttributes);
+        }
+
+        public static ObjectHandle CreateInstanceFrom(string assemblyFile, string typeName, object[] activationAttributes)
+        {
+            return CreateInstanceFrom(assemblyFile,
+                                      typeName,
+                                      false,
+                                      Activator.ConstructorDefault,
+                                      null,
+                                      null,
+                                      null,
+                                      activationAttributes);
+        }
+
+        private static ObjectHandle CreateInstanceFromInternal(String assemblyFile,
+                                                               String typeName,
+                                                               bool ignoreCase,
+                                                               BindingFlags bindingAttr,
+                                                               Binder binder,
+                                                               Object[] args,
+                                                               CultureInfo culture,
+                                                               Object[] activationAttributes)
+        {
+            Assembly assembly = Assembly.LoadFrom(assemblyFile);
+            Type t = assembly.GetType(typeName, true, ignoreCase);
+
+            Object o = Activator.CreateInstance(t,
+                                                bindingAttr,
+                                                binder,
+                                                args,
+                                                culture,
+                                                activationAttributes);
+
+            if (o == null)
+                return null;
+            else
```

Comment:
`else` is not needed here.

Reply:
i wanted to change here and in some other places...at the end i preferred be close as possible with old code.

Reply:
> else is not needed here.

sorry, you mean we'll return ObjectHandle with null(on unwrap) or you want only remove else?

Reply:
just remote the else

```
 if (o == null)
    return null;

return new ObjectHandle(o);
```

or even: `return (o != null) ? new ObjectHandle(o) : null;`

Reply:
>  the end i preferred be close as possible with old code.

We tend to reformat the code to match the current style for ports like these, especially when the change is small.

Reply:
understood